users/views.py

```diff
@@ -73,3 +68,45 @@ def resend_auth_mail(request, user_id):
     user.email_user('Account activation', 'Confirmation', html_message=html_body)
 
     return render(request, 'registration/login.html', context={'nextstep': True})
+
+
+def get_user_profile(request):
+    user = request.user
+    user_data = {
+        'Username': user.username,
+        'Email': user.email,
+        'Birthday': user.birthday,
+        'Password': None
+    }
+    context = {
+        'user': user,
+        'user_data': user_data,
+        'next': 'password_change_done',
+    }
+    return render(request, 'users/profile.html', context)
+
+
+def get_user_favourite(request):
+    game_list = get_games_id_list(request)
+    if game_list:
+        context = {
+            'games': game_list,
+        }
+    else:
+        context = {
+            'empty': 'There is no games here!',
+        }
```

Comment:
You should check for empty list in template and render this text accordingly there.